Cargo.toml

```diff
@@ -44,12 +44,16 @@ rand = "0.8.4"
 
 color-eyre = "0.5"
 
-near-crypto = "0.14"
-near-primitives = "0.14"
-near-jsonrpc-client = "0.4.0-beta.0"
-near-jsonrpc-primitives = "0.14"
+near-crypto = "0.15"
+near-primitives = "0.15"
+near-sdk = "4.0.0"
+near-jsonrpc-client = "0.4.1"
+near-jsonrpc-primitives = "0.15"
 
 interactive_clap = { git = "https://github.com/FroVolod/interactive-clap", rev = "23481402d7f36d9e7b0f5460f6d249bcf4c31ec5" }
 interactive_clap_derive = { git = "https://github.com/FroVolod/interactive-clap", rev = "23481402d7f36d9e7b0f5460f6d249bcf4c31ec5" }
 
 prettytable-rs = "^0.9"
+
+appchain-registry = { git = "https://github.com/octopus-network/octopus-appchain-registry", branch = "main" }
+appchain-anchor = { git = "https://github.com/octopus-network/octopus-appchain-anchor", branch = "v2.4.0" }
```

Comment:
The branch "v2.4.0" seems deleted, maybe need to use `tag="v2.4.0"`

Reply:
OK. Will fix.